jobs/germany-covid-forecast.sh

```diff
@@ -9,5 +9,8 @@ docker exec -w /home/rstudio/covid-german-forecasts forecast Rscript -e 'devtool
 # run forecasts
 docker exec -w /home/rstudio/covid-german-forecasts forecast bash rt-forecast/update.sh
 
+# redeploy rt crowd forecast app
+docker exec -w /home/rstudio/covid-german-forecasts forecast Rscript rt-crowd-forecast/redeploy-rt-forecast-app.R
```

Comment:
This doesn't change anything that gets committed right? If it doesn't I think putting it after the commit step makes sense.

Reply:
also whilst we are here I think renaming `redeploy-rt-forecast-app` to `redeploy-app` might save everyone some typing :)

Reply:
there is this .dcf file that gets commited as well. not sure we technically need it and could also add it to .gitignore I guess?